cypress/integration/smoke-tests/load-data-server-geoserver.spec.js

```diff
@@ -0,0 +1,124 @@
+/// <reference types="Cypress" />
+var os = require('../../support/selectors.js');
+
+describe('Add GeoServer', function() {
+  before('Login', function() {
+    cy.login();
+  });
+
+  it('Load data from GeoServer', function() {
+    // Add a server
+    cy.get(os.statusBar.SERVERS_BUTTON).click();
+    cy.get(os.settingsDialog.Panel.dataServers.ADD_SERVER_BUTTON).click();
+    cy.get(os.importURLDialog.ENTER_A_URL_FIELD)
+        .type('https://gdp-geoserver.dev.dev.east.paas.geointservices.io/geoserver/ows');
```

Comment:
Depending on third-party servers for tests is not going to be consistent.

Reply:
https://github.com/ngageoint/opensphere/issues/398